code/modules/clothing/under/_under.dm

```diff
@@ -23,6 +23,16 @@
 	var/mutantrace_variation = NO_MUTANTRACE_VARIATION //Are there special sprites for specific situations? Don't use this unless you need to.
 	var/freshly_laundered = FALSE
 
+/obj/item/clothing/under/Initialize(mapload)
+	. = ..()
+	if(random_sensor)
+		//make the sensor mode favor higher levels, except coords.
+		sensor_mode = pick(SENSOR_OFF, SENSOR_LIVING, SENSOR_LIVING, SENSOR_VITALS, SENSOR_VITALS, SENSOR_VITALS, SENSOR_COORDS, SENSOR_COORDS)
```

Comment:
why not a pick_weight :(

Reply:
I moved the init to the top of the file, the code is unedited.

Reply:
Actually I had to revert this in https://github.com/tgstation/tgstation/pull/65768/commits/4f7a30e391692b7a379f6f63b24cb7207d18d51c since all those defines are numbers instead of strings.